src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -566,6 +566,7 @@ protected override void Generate(RenameTableOperation operation, [CanBeNull] IMo
             Check.NotNull(builder, nameof(builder));
 
             var method = (string)operation[NpgsqlAnnotationNames.IndexMethod];
+            var operators = (string[])operation[NpgsqlAnnotationNames.IndexOperators];
```

Comment:
Any reason not to cast to `IReadOnlyList<string>`?

Reply:
No reason. Why would you?

Reply:
Hmm... I was reviewing this from my phone and must have misread something here. Please disregard.